src/edu/ucsd/library/xdre/web/MetadataImportController.java

```diff
@@ -70,40 +70,44 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
 				result = handler.execute();
 				if(result) {
 					List<String> soluctions = new ArrayList<>();
```

Comment:
overall :+1: 

should `soluctions` be `solutions` or `selections`? Note this for #L85 and that block as well.